gbfs.md

```diff
@@ -218,6 +326,41 @@ bikes             | Yes       | Array that contains one object per bike that is
 \- lon             | Yes       | Longitude of the bike. The field value must be a valid WGS 84 latitude in decimal degrees format. See: http://en.wikipedia.org/wiki/World_Geodetic_System, https://en.wikipedia.org/wiki/Decimal_degrees
 \- is_reserved     | Yes       | 1/0 value - is the bike currently reserved for someone else
 \- is_disabled     | Yes       | 1/0 value - is the bike currently disabled (broken)
+\- vehicle_type_id | Conditionally Required | The vehicle_type_id of this vehicle as described in [vehicle_types.json](#vehicle_typesjson). This field is required if the [vehicle_types.json](#vehicle_typesjson) is defined.
+\- last_reported | Optional       | Integer POSIX timestamp indicating the last time this vehicle reported its status to the backend
```

Comment:
Nit: "to the operator's backend", to make clear what backend this is.

Reply:
Ok, I'll add this.